src/convertToObject.js

```diff
@@ -12,7 +12,14 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesList = sourceString.split(';').filter(style =>
+    style.includes(':')).map(style => style.split(':'));
```

Comment:
Так читабельність зростає
```suggestion
  const stylesList = sourceString.split(';')
    .filter(style => style.includes(':'))
    .map(style => style.split(':'));
```